src/go/rdctl/pkg/shutdown/shutdown.go

```diff
@@ -117,7 +117,13 @@ func pkillQemu() error {
 	cmd.Stderr = os.Stderr
 	err := cmd.Run()
 	if err != nil {
-		return fmt.Errorf("failed to kill qemu: %w", err)
+		if exitError, ok := err.(*exec.ExitError); ok {
+			if exitCode := exitError.ExitCode(); exitCode != 0 || exitCode != 1 {
```

Comment:
`exitCode != 0 || exitCode != 1` is always true.
Using deMorgan: ~ => `!(exitCode == 0 && exitCode ==1)` => `!<always false>` => `true`

Given that, you'll need a comment here to explain what you're looking for

Reply:
Wow, totally missed that. Nice catch. I meant `exitCode != 0 && exitCode != 1`.